uk.co.compucorp.civicrm.hrleaveandabsences/views/admin-dashboard/components/admin-dashboard-container.html

```diff
@@ -23,3 +23,4 @@
     </ul>
   </div>
 </div>
+<div class="clear"></div>
```

Comment:
Could this be done without extra div? Is it possible to use something like `.admin-dashboard-page::after{ }`?